dpe/src/tci.rs

```diff
@@ -8,32 +8,18 @@ use zerocopy::AsBytes;
 #[cfg_attr(test, derive(zerocopy::FromBytes))]
 pub(crate) struct TciNodeData {
     pub tci_type: u32,
-
-    // Bits
-    // 31: INTERNAL
-    // 30-0: Reserved. Must be zero
-    flags: u32,
```

Comment:
Can you also update the spect to remove the flags field from TCI_NODE_DATA? I agree it's not needed.